newrelic-weaver/src/main/java/com/newrelic/weave/MethodCallInlinerAdapter.java

```diff
@@ -240,26 +240,17 @@ protected int newLocalMapping(final Type type) {
     }
 
     /**
-     * Flags method nodes requiring additional return insn processing, which for now is only invokeSuspend.
+     * Flags method nodes requiring additional return insn processing.
+     *
+     * It returns false by default.
+     *
+     * To enable this feature, set -Dnewrelic.config.class_transformer.clear_return_stacks=true
      * For future reference, if other code surfaces a bytecode verification failure (such as an ArrayIndexOutOfBoundsException),
-     * modify this guard to process additional methods beyond invokeSuspend.
+     * modify this guard to process additional methods.
      */
     private boolean shouldClearReturnStacks(String name, String desc) {
-        if (clearReturnStacksDisabled()) {
-            return false;
-        }
-        final String invokeSuspendName = "invokeSuspend";
-        final String invokeSuspendDesc = "(Ljava/lang/Object;)Ljava/lang/Object;";
-        return invokeSuspendName.equals(name) && invokeSuspendDesc.equals(desc);
-    }
-
-    /**
-     * Feature flag to disable return stack processing.
-     * To use this feature flag, set -Dnewrelic.config.class_transformer.clear_return_stacks=false at JVM startup.
-     */
-    private boolean clearReturnStacksDisabled() {
-        String enabled = System.getProperty("newrelic.config.class_transformer.clear_return_stacks", "true");
-        return enabled.equalsIgnoreCase("false");
+        String enabled = System.getProperty("newrelic.config.class_transformer.clear_return_stacks", "false");
+        return enabled.equalsIgnoreCase("true");
```

Comment:
This is perfectly fine, but you can also use `Boolean.getBoolean()` too.
https://docs.oracle.com/javase/8/docs/api/java/lang/Boolean.html#getBoolean-java.lang.String-

Reply:
yeah, let's do that

Reply:
Took your suggestion and inlined the method altogether since a whole method call is overkill now